hrui/js/src/hrui.js

```diff
@@ -140,8 +140,23 @@
    * Customizes the quick search field
    */
   function customizeQuickSearchField () {
-    toggleCustomClassToQuickSearchField();
     changeQuickSearchFieldPlaceholder();
+    manageCustomClassOfQuickSearchField();
+  }
+
+  /**
+   * Checks if the user has clicked outside of the quick search field
+   * by analyzing the given click target
+   *
+   * @param {object} target
+   * @return {boolean}
+   */
+  function hasUserClickedOutsideQuickSearchField (target) {
+    $target = $(target);
```

Comment:
Should be `var $target = ...`

I wonder why semistandard didn't yell? Or you can mutate `target = ...` to save some memory.

Reply:
wow, that's bad. I also forgot to enable the git hook in my local env, will do now